bin/runtime/src/migrations/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod scheduler;
+mod staking;
+
+pub use scheduler::MigrateToV3 as SchedulerMigrateToV3;
+pub use staking::BumpStorageVersionFromV7ToV10;
```

Comment:
I guess you can try to make the naming consistent. The approach for `Scheduler` is good.

Reply:
changed it in this pr